src/Orckestra.Composer.Search/Providers/BaseSearchCriteriaProvider.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Orckestra.Composer.Parameters;
+using Orckestra.Composer.Providers;
+using Orckestra.Composer.Search.RequestConstants;
+using Orckestra.Composer.Services;
+using System;
+using System.Threading.Tasks;
+
+namespace Orckestra.Composer.Search.Providers
+{
+    public class BaseSearchCriteriaProvider : IBaseSearchCriteriaProvider
+    {
+        protected IWebsiteContext WebsiteContext { get; private set; }
+        protected IInventoryLocationProvider InventoryLocationProvider { get; private set; }
+        protected IComposerContext ComposerContext { get; private set; }
+        protected IFulfillmentContext FulfillmentContext { get; private set; }
+
+        public BaseSearchCriteriaProvider(IWebsiteContext websiteContext, IInventoryLocationProvider inventoryLocationProvider, IComposerContext composerContext, IFulfillmentContext fulfillmentContext)
+        {
+            WebsiteContext = websiteContext ?? throw new ArgumentNullException(nameof(websiteContext)); ;
+            InventoryLocationProvider = inventoryLocationProvider ?? throw new ArgumentNullException(nameof(inventoryLocationProvider));
+            ComposerContext = composerContext ?? throw new ArgumentNullException(nameof(composerContext));
+            FulfillmentContext = fulfillmentContext ?? throw new ArgumentNullException(nameof(fulfillmentContext));
+        }
+
+        public virtual async Task<SearchCriteria> GetSearchCriteriaAsync(string searchTerms, int limit, int offset, string baseURL, bool includeFacets, int page, string sortBy, string sortDirection)
```

Comment:
please define new configuration
SearchConfiguration.DefaultSortDirection = "desc"
SearchConfiguration.DefaultSortBy = "score"

and use it by default everywhere.
Remove this SearchRequestParams.DefaultSortDirection and use new SearchConfiguration.DefaultSortDirection